README.md

```diff
@@ -16,54 +16,54 @@ Please submit all security-related bugs through Bugzilla using the [web security
 
 Never submit security-related bugs through a Github Issue or by email.
 
-  [bug bounty program]: https://www.mozilla.org/en-US/security/web-bug-bounty/
-  [FAQ pages]: https://www.mozilla.org/en-US/security/bug-bounty/faq-webapp/
-  [Bug Bounty Onramp page]: https://wiki.mozilla.org/Security/BugBountyOnramp/
-  [web security bug form]: https://bugzilla.mozilla.org/form.web.bounty
+[bug bounty program]: https://www.mozilla.org/en-US/security/web-bug-bounty/
+[faq pages]: https://www.mozilla.org/en-US/security/bug-bounty/faq-webapp/
+[bug bounty onramp page]: https://wiki.mozilla.org/Security/BugBountyOnramp/
+[web security bug form]: https://bugzilla.mozilla.org/form.web.bounty
 
 ## Requirements
 
-* You need [Node](https://nodejs.org/) 6.x which is the current
+- You need [Node](https://nodejs.org/) 6.x which is the current
   [LTS](https://github.com/nodejs/LTS) (long term support) release.
-* Install [yarn](https://yarnpkg.com/en/) to manage dependencies
+- Install [yarn](https://yarnpkg.com/en/) to manage dependencies
   and run scripts.
 
 The easiest way to manage multiple node versions in development is to use
 [nvm](https://github.com/creationix/nvm).
 
 ## Get started
 
-* type `yarn` to install all dependencies
-* type `yarn amo:stage` to start a local server that connects to a
+- type `yarn` to install all dependencies
+- type `yarn amo:stage` to start a local server that connects to a
   hosted staging server
 
 ## Development commands
 
 Here are some commands you can run:
 
-| Command                     | Description |
-|-----------------------------|-------------|
-| yarn amo                    | Start the dev server/proxy (for amo) using data from Docker |
-| yarn amo:dev                | Start the dev server/proxy (for amo) using data from the dev server (https://addons-dev.allizom.org/) |
-| yarn amo:no-proxy           | Start the dev server without a proxy (for amo) using data from Docker |
-| yarn amo:stage              | Start the dev server/proxy (for amo) using data from the staging server (https://addons.allizom.org/) |
+| Command                     | Description                                                                                                |
+| --------------------------- | ---------------------------------------------------------------------------------------------------------- |
```

Comment:
This entire table looks weird but I guess it renders ok

Reply:
Yeah, it renders correctly